docs/api-versioning.md

```diff
@@ -27,7 +28,7 @@ An API version with the label "prerelease" indicates that the version is not rea
 
 ### How Versions Are Incremented
 
-We currently only increment the major version whenever there is a breaking change which is considered to be not backwards compatible. All minor versions are implied to be `0`. All versions are in the format `Major.0`
+We currently only increment the major version whenever there is a breaking change which is considered to be not backwards compatible. All minor versions are implied to be `0`. All versions are in the format `Major.0`. Given the implicit nature of versions, just specifying `Major` version in the route would suffice.
```

Comment:
Let's just leave the first sentence talking about major version and remove everything else

Reply:
Done.